paymentsheet/src/main/java/com/stripe/android/paymentsheet/injection/PaymentSheetViewModelModule.kt

```diff
@@ -17,6 +24,24 @@ internal class PaymentSheetViewModelModule(private val starterArgs: PaymentSheet
         return starterArgs
     }
 
+    @Provides
+    fun providesIntentConfirmationHandlerFactory(
+        application: Application,
+        savedStateHandle: SavedStateHandle,
+        paymentConfigurationProvider: Provider<PaymentConfiguration>,
+        stripePaymentLauncherAssistedFactory: StripePaymentLauncherAssistedFactory,
+        intentConfirmationInterceptor: IntentConfirmationInterceptor,
+    ): IntentConfirmationHandler.Factory {
+        return IntentConfirmationHandler.Factory(
```

Comment:
If we do decide to keep the factory, it probably makes more sense to `@Inject` it's constructor.